modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -3,58 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To start monitoring Mule apps that are managed in CloudHub, you must deploy them to a supported version of Mule runtime engine and set the runtime version in your Mule app. 
+To start monitoring Mule apps that are managed in CloudHub, you must deploy them to a supported version of Mule runtime engine and set the runtime version in your Mule app.
 
 == Supported Versions of Mule Deployed to CloudHub
 
-Anypoint Monitoring non-AM images receive patch updates as soon as they are available, while -AM images receive the latest patch updates only once per month. It is recommended that you switch to a non-AM image if your production environment requires a Mule runtime engine patch.
+Anypoint Monitoring non-AM images receive patch updates as soon as they are available. Anypoint Monitoring -AM images are no longer receiving patch updates. It is recommended that you switch to a non-AM image if your production environment requires a Mule runtime engine patch.
 
-|===
-| Platform | Mule Version | Features
+Metrics and Logs are supported in the following Mule versions:
```

Comment:
```suggestion
Metrics and logs are supported in the following Mule versions:
```

Reply:
Be sure to GLOBALLY distinguish between init. capped proper nouns and lowercase common nouns.
See style reference.
When I am in doubt, I suggest lowercase; please let me know if a proper noun needs adding to the Term Presentation list.